molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl

```diff
@@ -85,9 +76,37 @@
     </#list>
 
     <#include "molgenis-header-tracking.ftl"><#-- before closing </head> tag -->
+    <script>
+        window.__STATE__ = {
```

Comment:
This is not necessary, we have different mechanism to fill this

Reply:
This was introduced to always have a context from Molgenis available in a frontend app, instead of app-specific __INITIAL_STATE__. This will be useful when we're splitting the frontend from the backend. Already used to detect the current theme

Reply:
Removed window.__STATE__and used GET '/api/data/sys_set_app/app' to get the appropriate settings in the theme manager